packages/launcher/style/base.css

```
@@ -141,8 +142,8 @@
 }
 
 .jp-LauncherCard[data-category='Notebook'] .jp-LauncherCard-noKernelIcon {
-  /* This color is copied from the notebook icon. */
-  color: #ef6c00;
+  /* This color is the Jupyter brand's one. */
+  color: #f37626;
```

Let's use the CSS property here:

```suggestion
  color: var(--jp-jupyter-icon-color);
```

---

> @HaudinFlorence would you mind not changing the launcher card styles in this PR?

Sorry for the changes committed by mistakes. They should have been removed in commit d8c9878

---

> Let's use the CSS property here:

This has been done in commit d8c9878 .